packages/typeahead/stories/Typeahead.stories.tsx

```diff
@@ -110,11 +109,8 @@ WithDisabledItem.args = {
   )
 };
 
-export const FilterWhileTyping = () => <FilteredListTypeahead items={items} />;
-
-export const CustomMenuEmptyState = () => (
-  <FilteredListTypeahead
-    menuEmptyState={<div className={padding("all")}>Nothing to show</div>}
-    items={items}
-  />
-);
+export const CustomMenuEmptyState = Template.bind({});
```

Comment:
This now only shows the empty state. I think the purpose of this story is to show an empty state when you type a value in the typeahead that has no matches against the available list. The other options should still be in the list though.

Reply:
yeah I guess that's what that helper was for. I could refactor it, but we basically have two stories that demonstrate how a helper works, not how the underlying component works. The behavior of the component is that when you send in empty items, it will display the custom menu state.

Reply:
okay I just reverted the changes and reintroduced the filtered state helper, refactored as a function. Took me only a few minutes and it's easier than making a decision about removing it